package.json

```diff
@@ -22,16 +22,15 @@
     "build:sassdoc": "sassdoc --config sassdoc.config.yaml ./src/govuk",
     "build:jsdoc": "typedoc",
     "build:app": "npm run build --workspace app",
-    "build:package": "gulp build:package",
-    "build:dist": "gulp build:dist",
+    "build:package": "gulp build:package --color",
```

Comment:
Gulp can't detect if the Terminal has support for colour when tasks from multiple workspaces are run together via `concurrently` or `npm-run-all`, but this also makes out GitHub Actions output look nicer